### DISCUSSION TITLE
DCDAO Constitution

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#58] [General]
# DCDAO Constitution

# Abstract

The DCDAO constitution is a set of binding rules that determine what governance actions are legitimate for the DAO.

All rules are equally binding. Actions taken under this Constitution may be on-chain or off-chain actions. On-chain actions are those that are actuated directly by the governance smart contracts of the DAO as transactions on a blockchain. Off-chain actions are those that are actuated by other means.

This Constitution also includes some "recommended guidelines" non-binding but strongly recommended as good governance practice.

This Constitution describes the procedures by which it may be amended.

# Membership and Voting Power

Every individual possessing a Profile NFT from the Darwinia Community DAO (DCDAO) is regarded as a member and can create proposals and cast votes on them.

The Profile NFT is a non-fungible token that represents a unique identity on the Darwinia Network. Its distribution involves three main steps: apply, audit, and release.

1. **Apply**
    
    To apply for a Profile NFT, a contributor must create a new application issue in the Profile GitHub repository. The issue should include the following information:
    
    - Contributor's public address on the Darwinia Network
    - Name and description of the profile
    - Selected image
    - Description of the contribution
2. **Audit**
    
    The audit process, is conducted by the Steering Committee. The audit verifies that the application meets the following requirements:
    
    - The contributor has a valid public address on the Darwinia Network
    - The profile name and description are appropriate and not misleading
    - Applicant’s contribution should be valid
    
    If a supermajority, constituting two-thirds of the Steering Committee votes in agreement, the application is deemed approved. And the votes must be recorded on the open public platform.
    
3. **Release**
    
    Once an application passes the audit process, the Profile NFT will be distributed to the contributor's public address on the Darwinia Network.
    
    The profile data is stored across two platforms: a GitHub repository and IPFS (InterPlanetary File System). The GitHub repository serves as the host for application issues and audit records, while IPFS is utilized for storing the Profile NFT metadata, which includes details such as the profile name, description, and image URL.
    

### Suggested Guidelines

Contributors to Darwinia Network are all qualified to apply for the Profile NFT and get the membership, eg:

- Developers, managers, and designers who have contributed to the development and maintenance of the network's underlying blockchain protocol, as well as developers who have built decentralized applications or tools on top of the network.
- Ambassadors who support and promote the adoption of the Darwinia Network through community engagement, education, and collaboration with project teams
- Moderators who are responsible for managing and supervising community operations
- Researchers who are exploring new applications and development trends of blockchain technology related to the Darwinia Network

### Voting Power

Voting power within the Darwinia Community DAO(DCDAO) is directly linked to the possession of a Profile NFT. Each Profile NFT equates to one vote, establishing a one-to-one relationship between NFT ownership and voting capability.

**Delegate Voting**: Members have the option to delegate their voting power to another trusted member. This allows for greater flexibility and representation, especially for members unable to participate actively in every vote.

# DAO Proposals and Voting Procedures

The following process governs the rules and procedures by which the DCDAO may propose, vote on, and implement DCDAO Proposals. No proposal may violate applicable laws, in particular sanctions-related regulations.

1. Phase 1: Temperature Check (1 week) (Optional but Recommended): The proposal is suggested on the [[public forum](https://github.com/orgs/dcdao/discussions)](https://github.com/orgs/dcdao/discussions) and discussed/debated for 1 week. A proposal that fails the temperature check should not be submitted for a vote. If a proposal fails the temperature check or has not undergone a temperature check, as a matter of good governance practice, it is recommended that voters strongly consider voting to reject it.
2. Phase 2: The proposal should be accompanied by a [[Snapshot](https://snapshot.org/#/darwiniacommunitydao.eth)](https://snapshot.org/#/darwiniacommunitydao.eth) poll or other method as determined under the governance process, which can only be submitted by any membership. The Snapshot poll also runs for at least 1 week and is decided by a simple majority with minimum membership voting participating meeting the quorum requirement. The quorum is at least 4 but can be updated to a larger number by passing new non-constitution proposals to change it as more members join.
3. Phase 3: Necessary on-chain and off-chain Execution.

Teams and executors who committed their duties to DCDAO through election, delegation, appointment, and proposals will need to execute the passed proposal according to their responsibilities.

Recommended guideline: These executors include but are not limited to committees, managers, Program participants, Proposal promise deliverers, and full-time or part-time positions. The executors of the proposals may assume the proposal can only enter the execution stage on the premise that the proposal is clear and explicit. When the content of the proposal is unclear or ambiguous, the executors have the right to refuse execution. Further proposals might be required to be proposed to resolve ambiguity.

1. Recommended guideline: DAO members should vote against any proposal that is incorrectly labeled.
    
    A Non-Constitutional proposal is not considered a "Constitutional proposal" including:
    
    - Funding: Requests funds/grants or proposes how to spend or allocate funds from the DAO Treasury.
    - Informational: Provides general guidelines or information to the community but does not otherwise propose a new or modified rule for the constitution.
    - Operational: Provides clean and clear information for the DCDAO team to operate but does not otherwise propose a new or modified rule for the constitution.
2. As a recommended guideline, a proposal should include:
    - Abstract - Two or three sentences that summarize the proposal.
    - Motivation - A statement on why the DCDAO team should support and execute the proposal.
    - Rationale - An explanation of how the proposal aligns with the DCDAO's mission and guiding values.
    - Key Terms (optional) - Definitions of any terms within the proposal that are unique, new to the DCDAO, and industry-specific.
    - Scope - A detailed breakdown of the scope of the proposal and the platform/technologies that will be used.
    - Steps to execute - The steps to execute the proposal, including associated costs, manpower, and other resources for each step where applicable. For the avoidance of doubt, any proposals involving transactions with third parties (such as grants) will need to ensure that applicable documentation and procedures are also included.
    - Timeline - Relevant timing details, including but not limited to start date, milestones, and completion dates.
    - Overall Cost - The total cost to implement the proposal.
3. As a recommended guideline, the proposal author can add additional fields to any template if necessary to fully communicate the intentions, specifics, and implications of the proposal.
    
    As a recommended guideline, resubmitted proposal should also include:
    
    - Here is a link to the original proposal;
    - Reasons such original proposal was not approved;
    - Changes that have been made and why it should now be approved; and
    - Any additional fields to any template if necessary to fully communicate the changes made and the intentions, specifics, and implications of such resubmitted proposals.
4. As a recommended guideline, DCDAO Treasury refers to the collective pool of funds or assets managed by a DCDAO. The treasury is a critical component of DCDAO, as it holds the financial resources that enable the DCDAO to function and pursue its objectives. There should be a multisig team for guarding and executing the proposals related to DAO Treasury.

# DAO’s **Management, Positions, and operation decision-making**

DCDAO decision-making is mainly by the Steering Committee. DCDAO is in its initial stages of formation, and the Steering Committee is appointed temporarily until a formal election process can be held. Members of this committee will regularly set up weekly meetings to review proposals and forum topics, coordinate and communicate resources, give suggestions for revision and improvement, and propose action guidelines. (Limit the number of members to <=5 to keep meetings efficient). DCDAO Steering Committee's working scope is as the following: 

1. Strategic Planning: Discussing and establishing the long-term goals, mission, and vision of the DCDAO.
2. Financial Support: Review the financial status, budget, and expenditures to ensure transparency and give suggestions respectively.
3. Governance Framework: Evaluating the governance structure and proposing changes or updates to ensure effective decision-making and participation from all stakeholders. Elect the DCDAO core members.
4. Decision-Making: Making important decisions that impact the organization, considering input from various stakeholders, and ensuring that the decisions align with the organization's values and objectives in the early stage.

**DAO Steering Committee Elections** 

Process for selecting and voting for the new cohort of Steering Committee members.

1. Nominee selection (10 days). Candidates must gain 3 DCDAO membership support to make it to the next step. There needs to be at least 5 nominees for the next round.
2. A compliance check by the DCDAO Community 10 days. The voice from the DCDAO community plays a critical role in fostering a collaborative and inclusive organizational culture, driving the non-profit DAO's mission forward, and ensuring that it remains responsive to the needs and expectations of its community members.
3. Member election (14 days). Votes are cast and the top 5 nominees are selected.

**New Membership Review**

DCDAO has clear criteria for evaluating new applications. This may include factors such as expertise in relevant domains, alignment with the DAO's values, and a commitment to actively contribute. By outlining these criteria, the review process becomes more objective, enabling members to assess applicants consistently. DCDAO's new membership application will be reviewed by the DCDAO steering committee monthly. 

**DAO Security Multisig Team Delegation**

The Darwinia Community DAO Multisig addresses are the digital wallets that have been established to manage the funds of the Darwinia Community DAO sponsored/donated by the Darwinia Treasury and Itering, this DAO has been created to support the development of the Darwinia ecosystem. The Multisig ensures that funds are managed securely and transparently, with multiple trusted members of the community required to sign off on any transactions before they can be executed. This helps to ensure that the interests of the Darwinia community are protected and that funds are used responsibly.

# Community Values

DCDAO aspires to become the most forward-thinking and action-orientated community-led organization with the dual objectives of supporting ecosystem growth and sustainably managing the treasury, and in doing so, maximizing our token value.

**Develop a thriving community**

DCDAO benefits from a vibrant ecosystem built on shared components and standards. Without the network effect of communities, the crypto space will be a graveyard of shiny ideas about a futuristic version of the internet.

**Operation transparency**

In DCDAO, all transactions, decisions, and rules are recorded on a public and immutable blockchain, providing an open and accessible ledger for anyone to inspect. This transparency ensures that all stakeholders have real-time visibility into the DAO's activities, including financial transactions, governance proposals, and voting outcomes. Participants can scrutinize the organization's operations, verify fund allocations, and monitor the overall health of the DAO.

# Changes and Amendments

Any change may be made to this constitution only by a two-thirds majority and at least 5 of all memberships participating.

The process for making amendments should be

1. Public: The amendment process should be open and transparent, allowing members and possibly the general public to be aware of the changes being proposed.

2. Notification to All Memberships: All members of the organization or body should be informed about the proposed amendments. This ensures that every member is allowed to understand and possibly contribute to the discussion regarding the changes.

3. Duration: The voting process should last at least 3 months. This duration ensures there is ample time for thorough discussion, debate, and consideration of the proposed amendments.

**Comment by SasoLithops** [comment]
Great job Cindy!
Will comment when I come back from holidays🎄

**Comment by SasoLithops** [comment]
Hi Cindy, sorry for late reply. Holidays got better of me.

So under **Membership and Voting Power**:

under _apply_ I would add questions I posted so that we get to know the member a little better.

under _suggested guidelines_ I would add for eg: 
- community members that are enthusiastic and are ready to help in discussions to push it forward, 
- Darwinia investors they tend to be more critical but bring another view that is good to be heard.

under _voting power_ WE MUST add quorum of 66% needs to be reached for proposal to be valid.



Under **DAO Proposals and Voting Procedures** 

I would do Phase one for at least one week on more important matters at least 2 weeks. As for Github being the best place to discuss this will be hard as many strongly against it. Maybe we could do it on Discord  first and then post to Github?

On Phase two one week is ok imho. But quorum must be 66% to be a decent DAO. And that members learn that participating in governance is to learn about DCDAO.


I ABSOLUTELY AGREE on _recommended guideline_  and anything else. Great job Cindy!!
